fiosdk/src/main/java/fiofoundation/io/fiosdk/FIOSDK.kt

```diff
@@ -3885,4 +4042,16 @@ class FIOSDK(private var privateKey: String, var publicKey: String,var technolog
 
         return Validator(isValid,if(!isValid) "Invalid Transfer Fio Address Request" else "")
     }
+
+    private fun validateAddBundledTransactions(fioAddress:String, bundleSets:Int, technologyPartnerId:String=""): Validator
+    {
+        var isValid = bundleSets > 0
+
+        isValid = isValid && fioAddress.isFioAddress()
+
+        if(technologyPartnerId.isNotEmpty())
+            isValid = isValid && technologyPartnerId.isFioAddress()
+
+        return Validator(isValid,if(!isValid) "Invalid Add Bundled Transactions Request" else "")
+    }
```

Comment:
Should we validate bundleSets, or just let the protocol handle error checking?

Reply:
we check that the bundle sets is >= 0. just to save a call to the protocol if its not appropriately set.

Reply:
Sounds good.